index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dany's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+                <a href="#"class="Logo">My Logo</a>
+                <nav class="Navbar">
+                    <div class="menu"><a href="#"><img src="img/humburger-btn.svg" alt="menu"></a></div>
+                </nav>
```

Comment:
- [ ] You have done a great job to ensure the hamburger icon is inside the `nav` tag as suggested by the previous reviewer 👏 👏 . However, we need to ensure that it matches the Figma design as much as possible. I suggest making sure it does not go outside the header. This is causing the portfolio to have a horizontal scroll. Kindly ensure that the horizontal scroll is also fixed ☺️ .

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/219882220-4c4cf32a-04ad-4c45-a74c-8962a8a6e042.png)